app/scripts/metamask-controller.js

```diff
@@ -2936,6 +2959,48 @@ export default class MetamaskController extends EventEmitter {
         },
         getAuthorizedScopesByOrigin,
       );
+      if (process.env.MULTICHAIN_API) {
+        this.controllerMessenger.subscribe(
+          `${this.accountsController.name}:selectedAccountChange`,
+          async (account) => {
+            if (
+              account.type === 'solana:data-account' &&
+              account.address !== lastSelectedSolanaAccountAddress
+            ) {
+              lastSelectedSolanaAccountAddress = account.address;
+
+              const originsWithSolanaAccountChangedNotifications =
+                getOriginsWithSessionProperty(
+                  this.permissionController.state,
+                  'solana_accountChanged_notifications',
```

Comment:
do we have a constant for this?

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/30949/commits/2ea1fcdcee5b15741ceaa19030caf1a8290a1c31